priv/ecto_repo/migrations/00000000000000_create_feature_flags_table.exs

```diff
@@ -1,8 +1,11 @@
 defmodule FunWithFlags.Dev.EctoRepo.Migrations.CreateFeatureFlagsTable do
   use Ecto.Migration
+  alias FunWithFlags.Config
+
+  @table_name Config.table_name() |> String.to_atom()
```

Comment:
I wouldn't make this dynamic in the migrations. The reason is that migrations should be as simple, logic-free, and dependency-free as possible. They should reflect the state of the DB at the moment in time the migration was created and run. Making it dynamic, here, would mean that the migration would have a different effect later, if run again after the config has changed.

I think adding a comment would be enough. Something that says that the table name can be optionally customized in the config, and that in that case the migration should reflect that.